src/filepicker.android.ts

```diff
@@ -93,18 +93,23 @@ function callIntent(context, intent, pickerType) {
 //     return uri.getAuthority() === 'com.google.android.apps.photos.content';
 // }
 
+/**
+ * Convert to android Array
+ */
+function convertToArray(types) {
```

Comment:
@alexist Can we cast the `types` to a string array for better autocompletion?

Reply:
And linting

Reply:
done for the cast. 
I haven't seen issue with lint, can you tell me what's wrong ?